design/invitation.go

```diff
@@ -38,6 +38,20 @@ var _ = a.Resource("invitation", func() {
 		a.Response(d.InternalServerError, JSONAPIErrors)
 		a.Response(d.BadRequest, JSONAPIErrors)
 	})
+
+	a.Action("rescindInvite", func() {
+		a.Security("jwt")
+		a.Routing(
+			a.DELETE("/rescind/:invitationID"),
```

Comment:
I would simply make this a DELETE on the `/invitations` endpoint (without the `/rescind` path).